website/ref/language.md

```diff
@@ -1664,21 +1664,36 @@ The following namespaces have special meanings to the language:
 
 ## Pre-Defined Modules
 
-Namespaces that are not special (i,e. one of the above) are also called
+Namespaces that are not special (i.e. one of the above) are also called
 **modules**. Aside from these special namespaces, Elvish also comes with the
-following modules:
+following modules that are always available, but might need to be imported by a
+`use` command:
```

Comment:
It is more productive to just list the names of all the pre-defined modules, as links to their respective reference pages, than repeating the index page of the reference section. It's also worthwhile to split out `edit` as a special case that might change in future.

Something like:

> * The following modules are always available: [exc:](exc.html), ...
> * The [unix](unix.html) module is available on UNIX-like platforms.
> * The [edit](edit.html) module is available in interactive module. As a special case, it does not need importing, but this may change in future.

Reply:
Fine by me but note the existing style of this list was created by you. :smile: Changed.